linode_api4/objects/profile.py

```diff
@@ -73,6 +100,11 @@ def enable_tfa(self):
     def confirm_tfa(self, code):
         """
         Confirms TFA for an account.  Needs a TFA code generated by enable_tfa
+
+        API Documentation: https://www.linode.com/docs/api/profile/#two-factor-authentication-confirmenable
+
```

Comment:
Looks like this is missing the `code` param

Reply:
I must have missed it. I'll add the missing param!